packages/fluentui/react-northstar-prototypes/src/prototypes/VirtualizedTree/VirtualTree.tsx

```diff
@@ -29,8 +29,8 @@ export interface VirtualTreeProps
       'selectedItemIds' | 'defaultSelectedItemIds' | 'onSelectedItemIdsChange' | 'selectable' | 'items'
     >,
     Pick<VariableSizeListProps, 'estimatedItemSize' | 'height'> {
-  // Where itemSize is in px
-  items?: ObjectShorthandCollection<TreeItemProps & { itemSize: number }>;
+  // Where 'item-size' is in px
+  items?: ObjectShorthandCollection<TreeItemProps & { 'item-size': number }>;
```

Comment:
Why this is required? 🤔

Reply:
You mean why 'item-size' is required? 
I saw in TMP this is how they pass the height for each item. So I did the same in my prototype. Because it is important to `resetAtIndex` when item expand/collapse, which clears the react-window item size cache. And I'd like to reflect this in my prototype